app-portage/eix/eix-0.32.1.ebuild

```diff
@@ -60,14 +73,22 @@ src_install() {
 }
 
 pkg_postinst() {
-	test -d "${EROOT}var/cache/${PN}" || {
-		mkdir "${EROOT}var/cache/${PN}"
-		use prefix || chown portage:portage "${EROOT}var/cache/${PN}"
-	}
-	local obs="${EROOT}var/cache/eix.previous"
-	! test -f "${obs}" || ewarn "Found obsolete ${obs}, please remove it"
+	local cache=${EROOT}var/cache/${PN}
```

Comment:
Well, ${EROOT}/var wouldn't have had much negative impact, whereas if we are going to remove the trailing slash we'll have to touch all the ${EROOT}var a second time ...

So yes, go for ${EROOT%/}/var please.